syntax/src/Language/Intentio/Parser.hs

```diff
@@ -70,6 +71,8 @@ parseExpr = M.parse expr
 
 mod :: Text -> Parser ModuleSource
 mod _moduleSourceName = do
+  _moduleExport      <- optional exportDecl
+  _moduleImports     <- many importDecl
```

Comment:
`_moduleExport`  and `_moduleImports` are not fields of the ModuleSource.

`_moduleSourceExport`, `_moduleSourceImport`